sdv/__init__.py

```diff
@@ -8,6 +8,89 @@
 __email__ = 'info@sdv.dev'
 __version__ = '1.1.1.dev0'
 
-from sdv._addons import _find_addons
 
-_find_addons('sdv_modules', globals())
+import sys
+import warnings
+from operator import attrgetter
+
+from pkg_resources import iter_entry_points
+
+import sdv.constraints
+import sdv.data_processing
+import sdv.datasets
+import sdv.evaluation
+import sdv.lite
+import sdv.metadata
+import sdv.metrics
+import sdv.multi_table
+import sdv.sampling
+import sdv.sequential
+import sdv.single_table
```

Comment:
Yeah I tried using `__all__`, but linting still fails and running `fix-lint` removes all the imports

Reply:
Gotcha, should we still add them to `__all__` anyway?

Reply:
If you do:
```python
from sdv import (
    constraints, data_processing, datasets, evaluation, lite, metadata, metrics, multi_table,
    sampling, sequential, single_table)

__all__ = (
    'constraints',
    'data_processing',
    'datasets',
    'evaluation',
    'lite',
    'metadata',
    'metrics',
    'multi_table',
    'sampling',
    'sequential',
    'single_table'
)
```
This will work, it does not detect them as used as they are not imported with that name. 

PS:
My first idea when I saw this was to use something like [this](https://docs.python.org/3/library/importlib.html#checking-if-a-module-can-be-imported) instead of importing the modules one by one. However the code above should work perfectly fine and it's more robust (I hope).